tests/test_files/dual_unknow.py

```
@@ -0,0 +1,159 @@
+_PKG_NAME = "fosslight_source"
+
+
+def main():
+    argv = sys.argv[1:]
+    path_to_scan = ""
+    write_json_file = False
+    output_file = ""
+    print_matched_text = False
+
+    try:
+        opts, args = getopt.getopt(argv, 'hmjp:o:')
```

Ruff (0.16.1)

[warning] Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)